testsuite/cluster-test/src/main.rs

```diff
@@ -474,7 +474,7 @@ impl ClusterTestRunner {
                 wait_committed: !args.burst,
             },
         };
-        let emit_to_validator = if cluster.fullnode_instances().is_empty() {
+        let emit_to_validator = if cluster.fullnode_instances().len() >= cluster.validator_instances().len() {
```

Comment:
probably just add another if else statement after the .is_empty() check:  cluster.fullnode_instances().len() == 1 {true}

Reply:
The updated condition should work for both cases.